.github/workflows/pr-ci.yml

```diff
@@ -73,6 +73,9 @@ jobs:
         if: env.SHOULD_RUN == 'true'
       - name: Regression test on affected projects
         run: tools/ci/run-chromatic-with-changed-projects.sh
+        env:
```

Comment:
This should probably be a separate commit since it isn't really related to popover

Reply:
True. I was merely using this pr opportunity to see how chromatic behaves. For a branch that only changes chromatic setting, nothing would get run (because no affected libs returned)

Reply:
Yeah I have no problem with it being part of this PR, just not the commit 👍